kpi/deployment_backends/kc_access/utils.py

```diff
@@ -70,7 +70,7 @@ def get_kc_profile_data(user_id):
         # The response will return a json.
         # We want the variable to have the same type in both cases.
         profile = profile_model.__dict__
-    except KobocatUserProfile.DoesNotExist:
+    except Exception:
```

Comment:
Why did we change `KobocatUserProfile.DoesNotExist` to `Exception` which seems different to what [kpi have](https://github.com/kobotoolbox/kpi/blob/fbc6b6b3609a22fc66f44cfe2af65b523cd705cb/kpi/deployment_backends/kc_access/utils.py#L72)?